backbone.js

```diff
@@ -70,6 +70,23 @@
 
   // Proxy Underscore methods to a Backbone class' prototype using a
   // particular attribute as the data argument
+  var modelMatcher = function(attrs) {
+    var matcher = _.matches(attrs);
+    return function(model) {
+      return matcher(model.attributes);
+    };
+  };
+  var modelProperty = function(key) {
+    return function(model) {
+      return model == null ? void 0 : model.get(key);
+    };
+  };
+  var cb = function(iteratee, instance) {
+    if (_.isFunction(iteratee)) return iteratee;
+    if (_.isObject(iteratee) && !instance._isModel(iteratee)) return modelMatcher(iteratee);
```

Comment:
it's cool to use `instance._isModel` here because instance will only ever be a Collection, right? (because `cb` is only used on 3- and 4-length methods, and modelMethods only uses 0- or 1-length).